### DISCUSSION TITLE
C File operations planned?

### DISCUSSION THREAD

**Discussion by adaumann** [drmortalwombat/oscar64#173] [Ideas]
Hi,
are there any plans for integration of File operations (read, write of Text and Binary Files). C typical fopen, fclose, fprintf etc. would be fine or something similar utilizing Kernal functions. In my next project I would like to have a save game functionality...

Andy

**Comment by drmortalwombat** [comment]
stdio type file operations are problematic with a 1541, there is no seek or read and write.  You can use the kernal functions with kernalio.h layer or use direct IEC communication with the iecbus.h

**Reply by adaumann** [reply]
I know that CC65 has partly implemented this, but I haven't looked into details
